src/libraries/System.Diagnostics.Tracing/ref/System.Diagnostics.Tracing.Counters.cs

```diff
@@ -11,26 +11,31 @@ public abstract partial class DiagnosticCounter : System.IDisposable
         public System.Diagnostics.Tracing.EventSource EventSource { get { throw null; } }
         public string Name { get { throw null; } }
         public void AddMetadata(string key, string? value) { }
+        [System.Runtime.Versioning.UnsupportedOSPlatform("browser")]
```

Comment:
Also, is there corresponding `src` code changes for this? I only see the `ref` changes.